projects/kserve/testing/config.yaml

```diff
@@ -102,7 +108,7 @@ ci_presets:
   quick_raw:
     extends: [raw, no_gpu, tgis]
 
-    tests.e2e.models: [flan-t5-small-hf-cpu, flan-t5-small-hf-cpu]
+    tests.e2e.models: [flan-t5-small-hf-cpu]
```

Comment:
please put the second model back before merging (having 2 recorded tests more code paths in the visualization)

Reply:
Replaced this